test_explicit_debug2.py

```
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+# Generated by Cribo - Python Source Bundler
+# https://github.com/ophidiarium/cribo
+
+# ─ Module: greetings.greeting ─
+message = "Hello"
+message = message
```

Pylint (3.3.7)

[warning] Assigning the same variable 'message' to itself

(W0127)